README.md

```diff
@@ -1,26 +1,127 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+```js
+//1. Descending, between "num" and 1.
+
+//while loop:
+let num = 10;
+while(num > 0) {
+  console.log(num)
+ num--
+};
+//for loop:
+for (let i = 10; i >= 1; i--){
+  console.log(i)
+};
+
+```
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+
+```js
+//2.Even or odd numbers
+
+//while loop:
+let num = 1;
+while(num <= 10) {
+ if(num % 2 === 0) {
+   console.log(num + " is even number.")
+ } else {
+    console.log(num + " is odd.")
+ }
+ num++
+};
+
+//for loop:
+for (let i = 1; i <= 10; i++){
+  if (i % 2 === 0){
+    console.log(i + ' is even')
+  } else {
+    console.log(i + ' is odd')
+  }
+};
+```
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
+```js
+/3. Iterates over all numbers
+
+//while loop:
+let num = 1
+while (num <= 10) {
+  console.log(num + " * 9 = " + num * 9);
+  num += 1
+};
+
+//for loop:
+for (let i = 1; i <= 10; i++){
+  console.log(i + " * 9 = " + i * 9);
+};
+
+```
+
+_Bonus_ think of another way to solve it.
   <details>
     <summary>
       Hint
     </summary>
     Find the final number and increment the loop by 9.
   </details>
-  
+
 4. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
 
+I DID BOTH TOGETHER~ 4 AND 5
 
 5. Modify your program to log "FizzBuzz", for numbers that are divisible by both 3 and 5 (still log "Fizz" or "Buzz" for numbers divisible by only one of those).
 
+```js
+//4. and 5.  Fizz & Buzz
+
+//while loop:
+let num = 1;
+	while (num <=100){
+	  if (num%3 && num%5 == 0){
+			console.log( + num + " FizzBuzz");
+		}
+		else if (num%3 == 0){
```

Comment:
please leave a space on either side when using the '%' operator. Also it's "===" not "=="

Reply:
Okay, always space between characters